taskcall/README.md

```diff
@@ -0,0 +1,33 @@
+# TaskCall
+
+## Overview
+
+TaskCall is a real-time incident response system that reduces system downtime by automating the response process. It continuously accepts feeds from monitoring tools to provide a comprehensive view of your system. It uses its on-call management and incident response mechanisms to assemble the right team and resolve incidents in the fastest possible time.
+
+TaskCall’s integration allows Datadog clients to bolster their operations by improving their incident awareness and simplifying the process in which they are handled. Incidents sync up bi-directionally between the two platforms. Once Datadog alerts are received in TaskCall, clients will be able to systematically manage all incidents without being interrupted. Clients will also benefit from improved impact visibility from dependency graphs and status dashboards. With better understanding of the state of your overall infrastructure efficient resolutions can be achieved.
+
+
+## Key Features
+
+- Correct on-call responders are notified as soon as an alert is received from Datadog.
+- Repeating alerts are automatically silenced to avoid interrupting the on-call responders when they are already looking at the incident.
+- The integration is bi-directional. Status and priority are synced up between Datadog and TaskCall.
+- Incident automatically resolves in TaskCall when alert conditions are not present anymore.
+- The integration is available on all TaskCall subscription plans.
+
+
+## Setup
+
+The integration needs to be configured from both TaskCall and Datadog. Please follow the [TaskCall Datadog Integration Guide][1] to set it up.
```

Comment:
Please add the instructions directly in this file, rather than linking to an external page. Additionally, please add an Uninstallation section as described [here](https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/#publish-the-oauth-client).